assets/wizards/popups/components/segment-group/SegmentGroup.test.js

```diff
@@ -0,0 +1,302 @@
+		campaign_groups: [],
+		categories: [],
+		content: 'Above Header Prompt 2',
+		id: 6,
+		options: {
+			placement: 'above_header',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Above Header Prompt 2',
+	},
+	{
+		campaign_groups: [],
+		categories: [
+			{
+				name: 'Tech',
+				term_id: 1,
+			},
+		],
+		content: 'Above Header Prompt 3 with category',
+		id: 7,
+		options: {
+			placement: 'above_header',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Above Header Prompt 3 with category',
+	},
+	{
+		campaign_groups: [],
+		categories: [
+			{
+				name: 'Tech',
+				term_id: 1,
+			},
+			{
+				name: 'News',
+				term_id: 2,
+			},
+		],
+		content: 'Above Header Prompt 4 with category',
+		id: 8,
+		options: {
+			placement: 'above_header',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Above Header Prompt 4 with category',
+	},
+	{
+		campaign_groups: [],
+		categories: [],
+		content: 'Custom Placement Prompt 1',
+		id: 9,
+		options: {
+			placement: 'custom1',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Custom Placement Prompt 1',
+	},
+	{
+		campaign_groups: [],
+		categories: [],
+		content: 'Custom Placement Prompt 2',
+		id: 10,
+		options: {
+			placement: 'custom1',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Custom Placement Prompt 2',
+	},
+	{
+		campaign_groups: [],
+		categories: [
+			{
+				name: 'Tech',
+				term_id: 1,
+			},
+		],
+		content: 'Custom Placement Prompt 3 with category',
+		id: 11,
+		options: {
+			placement: 'custom1',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Custom Placement Prompt 3 with category',
+	},
+	{
+		campaign_groups: [],
+		categories: [
+			{
+				name: 'Tech',
+				term_id: 1,
+			},
+			{
+				name: 'News',
+				term_id: 2,
+			},
+		],
+		content: 'Custom Placement Prompt 4 with category',
+		id: 12,
+		options: {
+			placement: 'custom1',
+			selected_segment_id: SEGMENT.id,
+			frequency: 'always',
+		},
+		status: 'publish',
+		title: 'Custom Placement Prompt 4 with category',
+	},
+];
+SEGMENT.prompts = PROMPTS;
+
+describe( 'A segment with conflicting prompts', () => {
+	const mockProps = {
+		campaignData: CAMPAIGN.campaignData,
+		campaign: CAMPAIGN.campaignId,
+		segment: SEGMENT,
+	};
+
+	beforeEach( () => {
+		// Mock global vars for custom placements.
+		window.newspack_popups_wizard_data = {
+			custom_placements: {
+				custom1: 'Custom Placement 1',
+			},
+		};
+
+		render(
+			<MemoryRouter>
+				<SegmentGroup { ...mockProps } />
+			</MemoryRouter>
+		);
+	} );
+
+	it( 'renders a conflict notice for uncategorized overlays in the same segment', async () => {
+		await waitForElement( () => screen.getByText( `Segment: ${ SEGMENT.label }` ) );
```

Comment:
Why should the tests await anything? There are no async action to wait for, the tests pass with these lines commented out.

Reply:
Removed in 01902e4b.